rel/overlay/etc/default.ini

```diff
@@ -57,6 +57,9 @@ backlog = 512
 docroot = {{fauxton_root}}
 socket_options = [{recbuf, 262144}, {sndbuf, 262144}, {nodelay, true}]
 require_valid_user = false
+; List of headers that will be kept when the header Prefer: return=minimal is included in a request.
+; If Server header is left out, Mochiweb will add its own one in.
+prefer_minimal = Cache-Control, Content-Length, Content-Type, ETag, Server, Vary
```

Comment:
we'll need `Transfer-Encoding` otherwise views, changes, _all_dbs will be broken.
we'll need `Set-Cookie` so we can login.
what about CORS related headers?
we'll need `Content-Range` for range reqs on attachments.
we'll need `WWW-Authenticate` for users that enable the standard login prompting setting.